### PR TITLE
[minor change] multicast address pools

### PR DESCRIPTION
**Pull request by akinross**
go client code for multicast address pools

### CODE REVIEW

**Review by sajagana — APPROVED** [review]
LGTM!

File: models/fvns_mcast_addr_blk.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+)
+
+const (
+	DnfvnsMcastAddrBlk        = "uni/infra/maddrns-%s/fromaddr-[%s]-toaddr-[%s]"
+	RnfvnsMcastAddrBlk        = "fromaddr-[%s]-toaddr-[%s]"
+	ParentDnfvnsMcastAddrBlk  = "uni/infra/maddrns-%s"
+	FvnsmcastaddrblkClassName = "fvnsMcastAddrBlk"
+)
+
+type AbstractionofIPAddressBlock struct {
```
Comment (lhercot) [inline models/fvns_mcast_addr_blk.go:17]: MulticastAddressBlock

Also change all the AbstractionofIPAddressBlock -> MulticastAddressBlock

File: client/fvnsMcastAddrBlk_service.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateAbstractionofIPAddressBlock(to string, _from string, multicast_address_pool string, description string, fvnsMcastAddrBlkAttr models.AbstractionofIPAddressBlockAttributes) (*models.AbstractionofIPAddressBlock, error) {
```
Comment (lhercot) [inline client/fvnsMcastAddrBlk_service.go:9]: CreateAbstractionofIPAddressBlock -> CreateMulticastAddressBlock 

Also adapt all other instance to similar format.

Comment (lhercot) [inline client/fvnsMcastAddrBlk_service.go:9]: why _from and not just from ?

Similar question for all other fct

**Review by lhercot — CHANGES_REQUESTED** [review]

File: models/fvns_mcast_addr_blk.go (hunk 1 above)
Comment (akinross) [reply models/fvns_mcast_addr_blk.go]: done

File: client/fvnsMcastAddrBlk_service.go (hunk 2 above)
Comment (akinross) [reply client/fvnsMcastAddrBlk_service.go]: done

Comment (akinross) [reply client/fvnsMcastAddrBlk_service.go]: done

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**